### ISSUE TITLE
Reaction rate limit issue from DDN not fixed

### ISSUE THREAD

**Issue by Nuyube** [Discord-Net-Labs/Discord.Net-Labs#26]
When you listen for reaction events, you'll eventually get your bot trying to delete a reaction which will repeat until you are temporarily banned from the gateway. This issue was fixed in DDN 2.4.0, but is still present in DNL.

**Comment by Nuyube** [comment]
```6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856537626717585488/reactions/%F0%9F%94%8D (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856537626717585488/reactions/%F0%9F%94%8D (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856489096015642644/reactions/3%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856499081475784704/reactions/0%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856499081475784704/reactions/1%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856499081475784704/reactions/2%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856477040181116948/reactions/0%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856476647078232074/reactions/1%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856455756633866260/reactions/%F0%9F%A9%B9 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856496331153342484/reactions/3%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856489096015642644/reactions/0%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856499081475784704/reactions/3%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856476647078232074/reactions/3%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856489096015642644/reactions/2%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856496331153342484/reactions/0%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856476647078232074/reactions/0%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856496331153342484/reactions/%E2%86%A9%EF%B8%8F (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856499081475784704/reactions/%E2%86%A9%EF%B8%8F (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:48 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856476647078232074/reactions/2%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:49 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856477040181116948/reactions/3%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:49 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856463358383751168/reactions/%E2%AC%86%EF%B8%8F (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:49 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856489096015642644/reactions/1%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:49 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856496331153342484/reactions/2%EF%B8%8F%E2%83%A3 (Bucket: d6b7697c78814ecd72bb20df05517c78)
6/21/2021 2:15:49 P|DETAIL  |Rest                            |Rate limit triggered: DELETE channels/826130259354451968/messages/856455756633866260/reactions/%E2%9C%A8 (Bucket: d6b7697c78814ecd72bb20df05517c78)```

There are no references to any functions that delete reactions.

**Comment by quinchs** [comment]
Once im able to again il merge upstream and fix the rest of the issues

**Comment by quinchs** [comment]
Fixed in [c6c4bc5](https://github.com/Discord-Net-Labs/Discord.Net-Labs/commit/c6c4bc57d2d273de7dfa5f930fb34f21a05d28f8)
